Makefile

```diff
@@ -1,37 +1,41 @@
-# mark targets to avoid confusion with files
-.PHONY: clean dir help java node
-all: clean java node
-
-JAVA_DIR = ./java/src/main/gen
-PROTO_DIR = ./api
+DIR = $(shell pwd)
+JAVA_DIR = ./java/src/main/java
 NODE_DIR = ./node
-PROTOC_BIN ?= protoc
-ALL_PROTO_FILES = $(shell find api -iname "*.proto" -exec echo -n '"{}" ' \; | tr '\n' ' ')
 
-dir: ## Creates outdir if necessary
-	[ -d $(JAVA_DIR) ] || mkdir -p $(JAVA_DIR)
+OUTPUT ?= $(JAVA_DIR)
+LANGUAGE ?= java
+GRPCPLUGIN ?= /usr/bin/protoc-gen-grpc-$(LANGUAGE)
+# with https://github.com/TheThingsIndustries/docker-protobuf/issues/9 we could use this image also for TypeScript
```

Comment:
Viel wichtiger: `grpc-node` fehlt

Reply:
I know, we could either provide that as a PR to `TheThingsIndustries/docker-protobuf` or build our own image